test/multiple_tracer_link_test.cpp

```diff
@@ -0,0 +1,3 @@
+// Contains nothing, but links in tracer_a.o and tracer_b.o to verify that
+// there's no multiple definition error from OpenTracingMakeTracerFactory.
```

Comment:
I wonder if this is guaranteed to cause a compilation error. Might be worth using the function pointer for some trivial operation to ensure the symbol isn't discarded during linking.

Reply:
Done.